luigi/metrics.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
```

Comment:
This appears to be a Python3.4+ module...

Please update to support 2.7 and 3.3.

Reply:
That's such a good point. I can either change how this work by removing the ENUM logic and replace it with something else that is retro-compatible. 

I suggest that we hold until #2278 is merged because that PR contains the change you're suggesting. I would have cherry-picked that specific commit, but the commit also includes changes that related directly to the PR, which I don't want to pull here.

I'll rebase once #2278 is merged and will re-run the spec suite and see how it goes from there! 

Thanks! :)